config/mvc-html-converter.php

```diff
@@ -0,0 +1,10 @@
+<?php
+
+return [
+    'default' => env('HTML_CONVERTER_SERVICE', 'PhantomMagickConverter'),
+    'providers' => [
```

Comment:
als je provider nog geen config verwacht zoals tokens of urls etc. dan mag dit weg.